example/main.dart

```diff
@@ -1,14 +1,14 @@
 import 'package:circular_buffer/circular_buffer.dart';
 
 main() async {
-  CircularBuffer<int> cb = new CircularBuffer<int>(5);
+  CircularBuffer<int> cb = new CircularBuffer(5);
```

Comment:
Use `final cb = CircularBuffer<int>(5);`

Reply:
I just changed it and you merged like 2 minutes earlier XD